internal/filesystem/filesystem.go

```diff
@@ -2313,6 +2315,9 @@ func (fs *FileSystem) runPlan(currentRunDir string, currentRunConfigPaths []stri
 	default:
 		logrus.Debug("ctx not closed")
 	}
+	if dumpCliLogs {
+		dumpLogFileToStdOut(currentRunDir, logFile)
```

Comment:
return error from the function and check the error here, logging if necessary

Reply:
I returned the error produced by `os.ReadFile(logFilePath)`